src/HasMedia/HasMediaTrait.php

```diff
@@ -94,6 +95,50 @@ public function addMediaFromUrl(string $url)
     }
 
     /**
+     * Add a base64 encoded file to the medialibrary.
+     *
+     * @param string $data
+     *
+     * @throws InvalidBase64Data
+     * @throws \Spatie\MediaLibrary\Exceptions\FileCannotBeAdded
+     *
+     * @return \Spatie\MediaLibrary\FileAdder\FileAdder
+     */
+    public function addMediaFromBase64(string $data)
+    {
+        if (! is_string($data)) {
```

Comment:
First time actually using scalar type hints.. so I did not think about this neat behaviour.. thanks!